SECURITY.md

```diff
@@ -44,7 +44,13 @@ You can also email the private [kubestellar-security-announce@googlegroups.com](
 
 ## Security Vulnerability Response
 
-Each report is acknowledged and analyzed by the maintainers of KubeStellar within 3 working days.
+Each report is acknowledged and analyzed by the maintainers of KubeStellar within 3 working days, which is well within the OpenSSF requirement of an initial response within 14 days.
+
+We guarantee that all valid vulnerability reports will receive an initial acknowledgment within 14 days of receipt.
```

Comment:
"We guarantee that" suggests a contractual relationship that does not exist. It is enough to state the intended behavior. As an open source project, we can not "guarantee" such a thing.